Parabolic/T/1D_dirichlet/line_60_heat.prj

```diff
@@ -0,0 +1,118 @@
+            <process_variables>
+                <process_variable>temperature</process_variable>
+            </process_variables>
+            <output>
+                <variables>
+                    <variable>temperature</variable>
+                    <variable>heat_flux_x</variable>
+                </variables>
+            </output>
+
+            <secondary_variables>
+                <heat_flux_x>heat_flux_x</heat_flux_x>
+            </secondary_variables>
+        </process>
+    </processes>
+
+    <parameters>
+        <parameter>
+            <name>K</name>
+            <type>Constant</type>
+            <value>3.2</value>
+        </parameter>
+        <parameter>
+            <name>Cp</name>
+            <type>Constant</type>
+            <value>1000</value>
+        </parameter>
+        <parameter>
+            <name>rho</name>
+            <type>Constant</type>
+            <value>2500</value>
+        </parameter>
+    </parameters>
+
+    <process_variables>
+        <process_variable>
+            <name>temperature</name>
+            <components>1</components>
+
+            <initial_condition>
+                <type>Uniform</type>
+                <values>0</values>
+            </initial_condition>
+
+            <boundary_conditions>
+                <boundary_condition>
+                    <geometrical_set>line_60_geometry</geometrical_set>
+                    <geometry>left</geometry>
+                    <type>UniformDirichlet</type>
+                    <value>1</value>
+                </boundary_condition>
+
+            </boundary_conditions>
+
+            <source_terms>
+            </source_terms>
+        </process_variable>
+    </process_variables>
+
+    <nonlinear_solvers>
+        <nonlinear_solver>
+            <name>basic_picard</name>
+            <type>Picard</type>
+            <tol>1e-6</tol>
+            <max_iter>10</max_iter>
+            <linear_solver>general_linear_solver</linear_solver>
+        </nonlinear_solver>
+    </nonlinear_solvers>
+
+    <linear_solvers>
+        <linear_solver>
+            <name>general_linear_solver</name>
+            <lis>-i cg -p jacobi -tol 1e-16 -maxiter 10000</lis>
+            <eigen>
+                <solver_type>CG</solver_type>
+                <precon_type>jacobi</precon_type>
+                <max_iteration_step>10000</max_iteration_step>
+                <error_tolerance>1e-16</error_tolerance>
+            </eigen>
+            <petsc>
+                <prefix>gw</prefix>
+                <parameters>-gw_ksp_type cg -gw_pc_type bjacobi -gw_ksp_rtol 1e-16 -gw_ksp_max_it 10000</parameters>
+            </petsc>
+        </linear_solver>
+    </linear_solvers>
+
+    <time_stepping>
+     <type>FixedTimeStepping</type>
+       <t_initial> 0.0 </t_initial>
+       <t_end> 39062500 </t_end>
+       <timesteps>
+         <pair><repeat>100</repeat><delta_t>390625</delta_t></pair>
```

Comment:
I have decrease the delta_t by factor 5, 25 and 125. The relative norm compared to analytical solution decrease from 1.6e-4 to 5.27e-5 and become stable around 4.72e-5. I would say that the reducing of the time step can result in a smoother solution but is not a key issue for this problem.